posthog-core/src/types.ts

```diff
@@ -26,6 +26,8 @@ export type PostHogCoreOptions = {
   fetchRetryDelay?: number
   /** Timeout in milliseconds for any calls. Defaults to 10 seconds. */
   requestTimeout?: number
+  /** Timeout in milliseconds for feature flag calls. Defaults to 3 seconds. */
+  featureFlagsRequestTimeoutMs?: number
```

Comment:
@neilkakkar the comment here should mention that for stateful is 10s

Reply:
oof nice catch thanks!